core/src/main/java/io/opencensus/stats/RpcMeasurementConstants.java

```diff
@@ -28,125 +23,125 @@
   public static final TagKey RPC_CLIENT_METHOD = TagKey.create("method");
   public static final TagKey RPC_SERVER_METHOD = TagKey.create("method");
 
-  // Constants used to define the following MeasurementDescriptors.
-  private static final List<BasicUnit> bytes = Arrays.asList(BasicUnit.BYTES);
-  private static final List<BasicUnit> scalar = Arrays.asList(BasicUnit.SCALAR);
-  private static final List<BasicUnit> seconds = Arrays.asList(BasicUnit.SECONDS);
+  // Constants used to define the following Measures.
+  private static final String BYTE = "By";
+  private static final String COUNT = "1";
+  private static final String MILLISECOND = "ms";
 
-  // RPC client {@link MeasurementDescriptor}s.
-  public static final MeasurementDescriptor RPC_CLIENT_ERROR_COUNT =
-      MeasurementDescriptor.create(
+  // RPC client {@link Measure}s.
+  public static final Measure RPC_CLIENT_ERROR_COUNT =
```

Comment:
You don't need to add a comment for all of them - just make a file-level comment to consider changing *_COUNT DoubleMeasures to LongMeasures